var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -42,6 +42,17 @@ def patch(self):
         rpath.append(os.path.join(
             self.prefix, pypkg.site_packages_dir, 'PySide'))
 
+        # Fix subprocess.mswindows check for Python 3.5
+        # https://github.com/pyside/pyside-setup/pull/55
+        filter_file(
+            '^if subprocess.mswindows:',
+            'mswindows = (sys.platform == "win32")\r\nif mswindows:',
+            "popenasync.py")
+        filter_file(
+            'subprocess.mswindows:',
```

Comment:
I'm not sure if you've actually changed anything here...

Reply:
@adamjstewart Sorry.  I don't understand what changes you are asking me to make.  Can you provide a diff or sample code to help me understand?

Reply:
I was thinking something like:
```suggestion
            '^    if subprocess.mswindows:',
```
or whatever it needs to be. Basically, I'm worried that your pattern might be too general, although I guess there's no reason you need to be too specific if you've tested all the versions in Spack.

Reply:
@adamjstewart I added the change that you suggested. Sorry about taking so long to do this.  

After I understood your concern I thought I would replace `filter_file` with a _patch_.  However, I couldn't formulate a patch file that would work since _pyside_'s sources use DOS line-endings and git apparently strips those from the patch file committed to spack.  The mismatch seemed to cause the patch process to fail.